### PR TITLE
Update to prod version 0.1.0

### PR DESCRIPTION
**Pull request by piotrwitkowski**
+ Allow injecting a http client

### CODE REVIEW

File: Duffel.ApiClient/Duffel.ApiClient.csproj [hunk 1]
```diff
@@ -6,13 +6,13 @@
         <Authors>Duffel Technology Ltd.</Authors>
         <Copyright>Duffel Technology Ltd.</Copyright>
         <Company>Duffel Technology Ltd.</Company>
-        <PackageVersion>0.0.3-alpha</PackageVersion>
+        <PackageVersion>0.0.4</PackageVersion>
```
Comment (sgerrand) [inline Duffel.ApiClient/Duffel.ApiClient.csproj:9]: Perhaps time to bump the minor version? 😺 
```suggestion
        <PackageVersion>0.1.0</PackageVersion>
```

File: Duffel.ApiClient/Duffel.ApiClient.csproj [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
         <Title>dotNet client for interacting with the Duffel API</Title>
         <PackageProjectUrl>https://duffel.com/docs/api/overview/welcome</PackageProjectUrl>
         <RepositoryUrl>https://github.com/duffelhq/duffel-api-dotnet</RepositoryUrl>
         <RepositoryType>Git</RepositoryType>
-        <AssemblyVersion>0.0.3</AssemblyVersion>
-        <FileVersion>0.0.3</FileVersion>
+        <AssemblyVersion>0.0.4</AssemblyVersion>
+        <FileVersion>0.0.4</FileVersion>
```
Comment (sgerrand) [inline Duffel.ApiClient/Duffel.ApiClient.csproj:15]:
```suggestion
        <AssemblyVersion>0.1.0</AssemblyVersion>
        <FileVersion>0.1.0</FileVersion>
```

**Review by sgerrand — COMMENTED** [review]
The functionality looks good to use "for real" now. 💪 

What do you think of using 0.1.0 as the release version?

File: Duffel.ApiClient/Duffel.ApiClient.csproj (hunk 1 above)
Comment (piotrwitkowski) [reply Duffel.ApiClient/Duffel.ApiClient.csproj:9]: Yes, I was thinking exactly that 👍

**Review by sgerrand — APPROVED** [review]
LGTM. 👍

**Pull request merged**